CHANGELOG.rst

```diff
@@ -2,6 +2,20 @@
 Changes
 =======
 
+1.x.x (TBD)
+-----------
+**Bugfixes and QOL**
+
+* [#66] updated commonground-api-common to 0.x.x (TBD)
```

Comment:
Should probably add a line to mention zgw-consumers being upgraded to 0.35.1 as well
```suggestion
* [#66] updated commonground-api-common to 2.1.0 (TBD)
```